src/components/slideshow/slideshow.js

```diff
@@ -145,10 +145,13 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
 
             var html = '';
 
-            if (options.interactive) {
+            html += '<div class="slideshowSwiperContainer"><div class="swiper-wrapper"></div></div>';
+
+            if (options.interactive && !layoutManager.tv) {
```

Comment:
So now we have no control.

Can we turn off key handling while OSD is visible (TV only)?

Reply:
We do. Left/right changes the slide, play starts the slideshow, back exits the slideshow.

I don't think having an OSD for remotes on this kind of slideshow is a good design, honestly.

Reply:
Yes. In this case, there is no need for a functional OSD. We just need a popup to show play/pause status.

But I am still trying to keep ability to use mouse on TV layout. Although YouTube app doesn't handle mouse, so maybe we shouldn't.

Reply:
I'm fully for mouse usage in the TV layout in the menus as well. For the image viewer, I think it'd hinder navigation more than anything.

I agree that a popup that would show up with a timeout to show the playback status is useful here.